scripts/build-cni-plugins

```diff
@@ -18,15 +18,37 @@
 
 set -ex
 
+architecture=""
+case $(uname -m) in
+    x86_64)
+	architecture="amd64"
+        ;;
+    aarch64)
```

Comment:
I get "arm64" for "uname -m" on my arm laptop, so maybe we can consider adding that as well